db/migrate/20170913110147_offences_workflows_in_escorts.rb

```diff
@@ -0,0 +1,8 @@
+class OffencesWorkflowsInEscorts < ActiveRecord::Migration[5.0]
+  def change
+    add_column :offences_workflows, :escort_id, :uuid
+    OffencesWorkflow.all.each { |ow| ow.update escort_id: Detainee.find(ow.detainee_id).escort_id }
+    remove_column :offences_workflows, :detainee_id, :uuid
+    add_index :offences_workflows, :escort_id
+  end
+end
```

Comment:
These sort of migrations can be dangerous.
1. Try and make them reversible
2. Consider writing output to show progress
3. Consider making this a one off rake task so it does not run as part of deploy - I think we took this approach a few times in PVB